zerver/migrations/0539_alter_stream_stream_topic_access_group.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 5.0.6 on 2024-05-17 19:07
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0538_set_default_for_stream_topic_access_group"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="stream",
+            name="stream_topic_access_group",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.RESTRICT,
+                related_name="+",
+                to="zerver.usergroup",
+            ),
+        ),
```

Comment:
The migration files should be added with the commit that add the fields in model and does other changes to make sure individual commit pass tests.